mgmtsystem/google.py

```diff
@@ -276,20 +276,30 @@ def handle_progressless_iter(error, progressless_iters):
         self.logger.info(json_dumps(response, indent=2))
         return (True, blob_name)
 
-    def create_vm(self, instance_name, source_disk_image=None, machine_type=None,
-            startup_script_data=None, timeout=180):
-        if self.does_vm_exist(instance_name):
-            self.logger.info("The {} instance is already exists, skipping".format(instance_name))
-            return True
+    def create_vm(self):
+        raise NotImplementedError('create_vm not implemented.')
```

Comment:
In the ideal world, deploy_template would call create_vm with the right parameters, because that would make the most sense but we *don't* do it in other places so this is just a rant of software idealist `</joke>` :smile: